browser/scripts/ui-site.js

```diff
@@ -84,7 +87,7 @@ var siteUI = new function() {
 		VizorUI.enablePopupEmbedLinks($body);
 
 		var signupCallback = function() {	// slush
-			window.location.href = '/edit';
```

Comment:
@kschzt  yes, this is the fallback behaviour if the form is found on a static page (which it normally isn't), so only a fail-safe measure. various pages handle this differently with most of them actually going to /edit like before.